programmer-skills/markdown-cheat-sheet.md

```diff
@@ -96,7 +96,90 @@ The number of hashtags indicates what kind of header you are creating.
 
 repeat(foo, 5);
  ```
+* Tables
+ * Each header cell must be seperated by at least three dashes.  
+ * The outer pipes ('|') are optional.
+ * [Mardown table generator](http://www.tablesgenerator.com/markdown_tables) is a useful tool to make your life easier.
+ 
+ ```
+|    table      |     example   |  :D   |     
+| ------------- |:-------------:| -----:|
```

Comment:
It would be good if you spell out how left / centre / right-aligning is achieved, as it's not too obvious at the moment